File: C++/isPrime.cc
#include <iostream>
#include <cmath>

using std::cout;

int isPrime(int num) {
    const int LARGEST_NUM_TO_SEARCH = (int) floor(sqrt(num));
    for (int i = 0;i < 10;i++) {
        cout << 10 % i;
    }
    return 1;
}
int mod(int x, int y) {
    return x % y;
}
int main() {
    isPrime(12);
    return 0;
}

File: TS/isPrime.ts
function isPrime(num: number): boolean {
    if (num == 1)
        return false
    else if (num > 1 && num <= 3)
        return true
    const largestNumToCheck = Math.floor(Math.sqrt(num));
    for (var i = 0;i < largestNumToCheck;i++) 
        if (num % i == 0)
            return false;
    return true;
}

var num = 4012931;
console.log(`Is ${num} a prime number? ${isPrime(num) ? "yes" : "no"}`);